Snippets/Rabbit/Rabbit_4/Usage.cs

```diff
@@ -92,7 +92,7 @@ void UseRoutingTopology(EndpointConfiguration endpointConfiguration)
         #region rabbitmq-config-useroutingtopology
 
         var transport = endpointConfiguration.UseTransport<RabbitMQTransport>();
-        transport.UseRoutingTopology<MyRoutingTopology>();
+        transport.UseRoutingTopology(durableExchangesAndQueues => new MyRoutingTopology());
```

Comment:
OK, I can do that. The only problem is that the compilation of snippets throws a compiler error on the obsolete, since warnings are reported as errors. How should I handle that? Should I use a local `#pragma warning disable CS0618`?